blocks/library/gallery/index.js

```diff
@@ -111,18 +93,29 @@ registerBlockType( 'core/gallery', {
 						onChange={ updateAlignment }
 					/>
 					{ !! images.length && (
-						<Toolbar controls={ [ {
-							icon: 'edit',
-							title: __( 'Edit Gallery' ),
-							onClick: () => editMediaLibrary( attributes, setAttributes ),
-						} ] } />
+						<Toolbar>
+							<li>
+								<MediaUploadButton
+									buttonProps={ {
+										className: 'components-icon-button components-toolbar__control',
+										'aria-label': __( 'Edit Gallery' ),
+									} }
+									onSelect={ onSelectImages }
+									type="image"
+									multiple="true"
+									gallery="true"
```

Comment:
Can't we avoid the `="true"` for these two props: multiple and gallery, seems like boolean props to me?